cfgov/jinja2/v1/_layouts/base-base.html

```diff
@@ -0,0 +1,245 @@
+    <style>
+        .optimize-loading { opacity: 0 !important; }
+    </style>
+    <script>
+        (function(a,s,y,n,c,h,i,d,e){s.className+=' '+y;h.start=1*new Date;
+        h.end=i=function(){s.className=s.className.replace(RegExp(' ?'+y),'')};
+        (a[n]=a[n]||[]).hide=h;setTimeout(function(){i();h.end=null},c);h.timeout=c;
+        })(window,document.documentElement,'optimize-loading','dataLayer',5000,
+        {'GTM-KHB8MB':true});
+    </script>
+    {# end Google Optimize page-hiding snippet #}
+
+    {# Google Analytics/Optimize snippet #}
+    {#
+      The last line in the script requires Optimize.
+      Pageview call removed, because pagviews are sent to Analytics via
+      the Google Tag Manager tag (code included below).
+    #}
+    <script>
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+        })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+        ga('create', 'UA-54439736-1', 'auto');
+        ga('require', 'GTM-KHB8MB');
+    </script>
+    {# end Google Analytics/Optimize snippet #}
+    {% endif %}
+
+    {# Google Tag Manager #}
+    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer','GTM-KMMLRS');</script>
+    {# end Google Tag Manager #}
+
+    {# Customized Modernizr build that includes html5printshiv.
+       Built via gulp-modernizer in `scripts.js` task.
+       TODO: Determine if modernizr is actually needed! #}
+    {% block modernizr %}
+    <script>
+        {% include '/js/modernizr.min.js' %}
+    </script>
+
+    {#
+      Turn off JavaScript for browsers that don't support ECMAScript 5 features
+      (e.g. Internet Explorer 8)
+      by reversing no-js/js CSS class change made by modernizr.
+      The following ECMAScript 5 feature checks come from
+      https://github.com/Modernizr/Modernizr/tree/master/feature-detects/es5
+    #}
+    <script>
+        var modernizr = window.Modernizr;
+        if ( !( typeof modernizr !== 'undefined' &&
+             modernizr.es5array &&
+             modernizr.es5date &&
+             modernizr.es5function &&
+             modernizr.es5object &&
+             modernizr.json &&
+             modernizr.es5string &&
+             modernizr.es5syntax &&
+             modernizr.es5undefined ) ) {
+          var docElement = document.documentElement;
+          docElement.className = docElement.className.replace( /(^|\s)js(\s|$)/, '$1no-js$2' );
+        }
+    </script>
+    {% endblock %}
```

Comment:
Please update this to `{% endblock modernizr %}` for consistency with most of the other non-inline `endblock` tags in this template.